packages/lab/src/layout/utils/useChangeView.ts

```diff
@@ -4,15 +4,15 @@ import {
   useOrderedBreakpoints,
 } from "@jpmorganchase/uitk-core";
 
-export const useIsStacked = (stackedAtBreakpoint: keyof Breakpoints) => {
+export const useChangeView = (stackedAtBreakpoint: keyof Breakpoints) => {
```

Comment:
For me the name  `useChangeView` isn't super intuitive as to what this hook is doing. I guess the hook tells us if the viewport is larger than the breakpoint you're querying so maybe something like `useIsViewportLargerThanBreakpoint` (but something better if you can think of it! 😊 ).

We should also update the name of `stackedAtBreakpoint` argument